esp/xslt/esdl2monitor.xslt

```diff
@@ -203,32 +204,47 @@ END;
     string100 Message {xpath('Message')};
   END;
 
-  soapoutRec := record
+  soapoutRec := record <xsl:choose><xsl:when test="$platform='ESP'">(the_responseLayout)
```

Comment:
I might be able to play tricks but then the xslt code is not as intuitive in the way it shows what the generated code is going to look like.